PCAxis.Serializers/JsonStat2/Model/ExtensionDimension.cs

```diff
@@ -92,6 +86,40 @@ public class ExtensionDimension : IEquatable<ExtensionDimension>
         [DataMember(Name = "lastPeriod", EmitDefaultValue = false)]
         public string LastPeriod { get; set; }
 
+        /// <summary>
+        /// Gets or Sets TimeUnit
+        /// </summary>
+        [DataMember(Name = "timeUnit", EmitDefaultValue = false)]
```

Comment:
I'm having second thoughts about forcing the EmitDefaultValue in PCAxis.Serializers

Reply:
Maybe we should make it nullable https://github.com/PxTools/PxApiSpecs/pull/125

Reply:
I've tried making it work but, `boolean` and `enum` always gets `EmitDefaultValue = true`